> I'd appreciate if somebody could take a look at the win-64 test failures. The error only ever happens with the first test to call `tkinter`, so each time I skip a failing test, it just causes the same error to crop up on another test.

The error is:
```
>       self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)
E       _tkinter.TclError: invalid command name "tcl_findLibrary"
..\_test_env\lib\tkinter\__init__.py:2270: TclError
```
~~Does that mean that tkinter is picking up the wrong Tcl/Tk library files? What if you set the `TCL_LIBRARY` and `TK_LIBRARY` environment variables? Or is there an issue with `matplotlib`?~~

Hm, see https://github.com/python/cpython/blob/v3.9.19/Lib/tkinter/__init__.py#L37. It fails for **python 3.9.19**. Is it wrongly configured or built?

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -46,7 +46,7 @@ test:
   commands:
     - pip check
     - pytest arviz/tests --ignore="arviz/tests/external_tests" -k "not (test_io_function or test_io_method or test_warning_rc_file or test_bad_rc_file or test_deterministic or test_empty_inference_data_object or test_rctemplate_updated or test_rc_context_file)"  # [not win]
-    - pytest arviz/tests --ignore="arviz/tests/external_tests" --ignore="arviz/tests/base_tests/test_plots_bokeh.py" --ignore="arviz/tests/base_tests/test_plots_matplotlib.py -k "not (test_io_function or test_io_method or test_warning_rc_file or test_bad_rc_file or test_deterministic or test_empty_inference_data_object or test_rctemplate_updated or test_rc_context_file or test_plot_forest_single_value or test_plot_density_discrete_combinedims or test_plot_violin or test_plot_density_float)"  # [win]
+    - pytest arviz/tests --ignore="arviz/tests/external_tests" --ignore="arviz/tests/base_tests/test_plots_bokeh.py" --ignore="arviz/tests/base_tests/test_plots_matplotlib.py" -k "not (test_io_function or test_io_method or test_warning_rc_file or test_bad_rc_file or test_deterministic or test_empty_inference_data_object or test_rctemplate_updated or test_rc_context_file or test_plot_forest_single_value or test_plot_density_discrete_combinedims or test_plot_violin or test_plot_density_float)"  # [win]
 
 about:
   home: https://github.com/arviz-devs/arviz

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -45,8 +45,8 @@ test:
     - bokeh >=1.4.0
   commands:
     - pip check
-    - pytest arviz/tests --ignore="arviz/tests/external_tests" -k "not (test_io_function or test_io_method or test_warning_rc_file or test_bad_rc_file or test_deterministic or test_empty_inference_data_object or test_rctemplate_updated or test_rc_context_file)"  # [not win or (win and py>39)]
-    #- pytest arviz/tests --ignore="arviz/tests/external_tests" -k "not (test_io_function or test_io_method or test_warning_rc_file or test_bad_rc_file or test_deterministic or test_empty_inference_data_object or test_rctemplate_updated or test_rc_context_file or test_plot_forest_single_value or test_plot_density_discrete_combinedims or test_plot_violin or test_plot_density_float)"  # [win]
+    - pytest arviz/tests --ignore="arviz/tests/external_tests" -k "not (test_io_function or test_io_method or test_warning_rc_file or test_bad_rc_file or test_deterministic or test_empty_inference_data_object or test_rctemplate_updated or test_rc_context_file)"  # [not win]
+    - pytest arviz/tests --ignore="arviz/tests/external_tests" --ignore="arviz/tests/base_tests/test_plots_bokeh.py" --ignore="arviz/tests/base_tests/test_plots_matplotlib.py -k "not (test_io_function or test_io_method or test_warning_rc_file or test_bad_rc_file or test_deterministic or test_empty_inference_data_object or test_rctemplate_updated or test_rc_context_file or test_plot_forest_single_value or test_plot_density_discrete_combinedims or test_plot_violin or test_plot_density_float)"  # [win]
 
 about:
   home: https://github.com/arviz-devs/arviz

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -45,8 +45,8 @@ test:
     - bokeh >=1.4.0
   commands:
     - pip check
-    - pytest arviz/tests --ignore="arviz/tests/external_tests" -k "not (test_io_function or test_io_method or test_warning_rc_file or test_bad_rc_file or test_deterministic or test_empty_inference_data_object or test_rctemplate_updated or test_rc_context_file)"  # [not win]
-    - pytest arviz/tests --ignore="arviz/tests/external_tests" -k "not (test_io_function or test_io_method or test_warning_rc_file or test_bad_rc_file or test_deterministic or test_empty_inference_data_object or test_rctemplate_updated or test_rc_context_file or test_plot_forest_single_value or test_plot_density_discrete_combinedims or test_plot_violin or test_plot_density_float)"  # [win]
+    - pytest arviz/tests --ignore="arviz/tests/external_tests" -k "not (test_io_function or test_io_method or test_warning_rc_file or test_bad_rc_file or test_deterministic or test_empty_inference_data_object or test_rctemplate_updated or test_rc_context_file)"  # [not win or (win and py>39)]
+    #- pytest arviz/tests --ignore="arviz/tests/external_tests" -k "not (test_io_function or test_io_method or test_warning_rc_file or test_bad_rc_file or test_deterministic or test_empty_inference_data_object or test_rctemplate_updated or test_rc_context_file or test_plot_forest_single_value or test_plot_density_discrete_combinedims or test_plot_violin or test_plot_density_float)"  # [win]
 
 about:
   home: https://github.com/arviz-devs/arviz

